### ISSUE TITLE
[turborepo] Windows VSCode terminal issues with Turbo UI

### ISSUE THREAD

**Issue by chris-olszewski** [vercel/turborepo#7808]
in my windows vscode terminal, pressing the arrow key moves 2 packages at a time, and pressing enter does nothing

https://github.com/vercel/turbo/assets/74945038/29fc34e2-8259-40cc-95da-cec7b7bdddad

also pressing ctrl+c keeps the logs in the same place 
![image](https://github.com/vercel/turbo/assets/74945038/66a8fddf-92f9-4ce6-8209-1b07c9c5c933)

_Originally posted by @imranbarbhuiya in https://github.com/vercel/turbo/discussions/7802#discussioncomment-8868920_

**Comment by chris-olszewski** [comment]
Reopening as PR didn't address the screen not clearing.

**Comment by chris-olszewski** [comment]
@imranbarbhuiya I've been unable to reproduce the final two issues you reported. Can you update to the latest canary and verify the following are still happening?
 - Screen isn't cleared on `Ctrl-C` press? Are you just pressing `Ctrl-C` once?
 - Enter doesn't allow for a task to be interacted with: Is the task you're trying to interact with marked with either [`persistent`](https://turbo.build/repo/docs/reference/configuration#persistent) or `interactive`?

If both issues are persisting, can you include your Windows build version?

**Comment by imranbarbhuiya** [comment]
https://github.com/vercel/turbo/assets/74945038/1c405607-eb0f-48b6-881f-75e68f8644f9

updated to canary, the arrow and enter issues are fixed now but the `ctrl+c` issue is still there. And yes I'm pressing `ctrl+c` only once but it prints `^C^C`, also seems like a new issue got introduced. Check the above recording, after running the command, the package names get hidden. But when I press the arrow keys, it reappears.

my windows infos are

```sh
Edition	Windows 11 Pro
Version	23H2
Installed on	‎10-‎07-‎2023
OS build	22635.3350
Experience	Windows Feature Experience Pack 1000.22688.1001.0
```

**Comment by Kaweees** [comment]
I get the same issue on macOS when using this templatehttps://github.com/dan5py/turborepo-shadcn-ui

**Comment by GeorgeHulpoi** [comment]
I have a similar problem with Windows 10. while I'm in interactive mode on any task, I can't navigate (scroll) through logs.

**Comment by anthonyshew** [comment]
We've fixed a ton of issues with the TUI since last activity here and I can't reproduce any of the problems stated above. If you run into any problems with the TUI, please open a new Issue with as much reproduction/example information as you can.
